packages/contracts/docs/osx/01-how-it-works/02-framework/02-plugin-management/01-plugin-repo/01-plugin-repo-creation.md

```diff
@@ -2,7 +2,7 @@
 title: Publishing a Plugin
 ---
 
-## The Plugin Repo Creation Process
+## To Create a PluginRepo is to Publish a Plugin
```

Comment:
Not sure I follow this one

Reply:
Ok, it personally creates a bit of confusion the fact of using capitalized words. Things like `Plugin` or `PluginRepo` are the exact names of a contract, whereas `Create` or `Publish` are standard words. 

Additionally, publishing a plugin involves two steps: creating your PluginRepo and registering it on the PluginRegistry.

Some suggestions for clarity:
- `Create a PluginRepo to publish your Plugin`
- `Start publishing your Plugin by creating a PluginRepo`
- `Publishing a Plugin is the same as creating a PluginRepo`
  -  _(I don't like it, because we miss the second step required)_